packages/terra-modal/src/ModalOverlay.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { PropTypes } from 'react';
+import './ModalOverlay.scss';
+
+const propTypes = {
+  classNameOverlay: PropTypes.string,
+};
+
+const defaultProps = {
+  classNameOverlay: null,
+};
+
+const ModalOverlay = ({ classNameOverlay, ...customProps }) => <div className={classNameOverlay} {...customProps} />;
```

Comment:
I kept this logic separate in case we want to create an overlay component. That way it would be easy to switch out.